addon/commands/insert-table-row-command.ts

```diff
@@ -1,49 +1,39 @@
-import Command from "./command";
+import Command from "@lblod/ember-rdfa-editor/commands/command";
 import Model from "@lblod/ember-rdfa-editor/model/model";
 import ModelSelection from "@lblod/ember-rdfa-editor/model/model-selection";
-import ModelTable from "@lblod/ember-rdfa-editor/model/model-table";
 import {MisbehavedSelectionError} from "@lblod/ember-rdfa-editor/utils/errors";
-import {logExecute} from "@lblod/ember-rdfa-editor/utils/logging-utils";
-
+import ModelTable from "@lblod/ember-rdfa-editor/model/model-table";
 
-export default class InsertTableColumnBeforeCommand extends Command {
-  name = "insert-table-column-before";
+export default class InsertTableRowCommand extends Command {
+  name = "insert-table-row";
 
   constructor(model: Model) {
     super(model);
   }
 
-  canExecute(): boolean {
-    return true;
-  }
-
-  @logExecute
-  execute(): void {
-
-    const selection= this.model.selection;
+  execute(above: boolean, selection: ModelSelection = this.model.selection) {
```

Comment:
using a boolean to determine behavior is a bit of a code smell IMHO. I find that usually it's better to have two differently named methods in such cases. In this case I think that translates most to having two separate commands. Giving the specific behavior its own name makes it easier to look for the specific usage later on, it's usually quite hard to search for a method with a certain param value (though some IDE's can do that for you)

Reply:
That's a good point. I wasn't too sure about this either. I gave the insert column before and after + row above and below their own classes again, that make use of the abstraction made earlier.